config.yml.example

```diff
@@ -0,0 +1,7 @@
+login:
+    email: me@example.com
+    password: secretpassword
+
+data:
```

Comment:
In the future, when we support following many athletes, I think we can make this a list, e.g.:

```
athletes:
  - 12345
  - 67890
...
```